### DISCUSSION TITLE
close windows left

### DISCUSSION THREAD

**Discussion by mortifia** [dbeaver/dbeaver#13130] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
add an option to have close windows to left 
![image](https://user-images.githubusercontent.com/22671054/123400650-48f16880-d5a6-11eb-9b0c-1a5db6dce66c.png)

**Comment by uslss** [comment, marked as answer]
it is already implemented in tab context menu. Does it solve your issue?
![image](https://user-images.githubusercontent.com/31996417/123610052-e6e46d80-d808-11eb-9ad4-43c6f63fe634.png)

**Comment by mortifia** [comment]
This option close all tab in left or right

I search to have close icon to left

![Capture](https://user-images.githubusercontent.com/22671054/125164734-ff9f3c80-e193-11eb-8edd-6fa96541999b.PNG)
